index.js

```diff
@@ -1,32 +1,30 @@
-import React, { PureComponent } from 'react';
-import PropTypes from 'prop-types';
-import { WebView } from 'react-native-webview';
+import React, { PureComponent } from "react";
```

Comment:
@tommasini Think we can revert these unrelated style/formatting-changes like changing quotes out of this PR and keep it to the relevant change? Will make a difference when syncing this fork with upstream.

Reply:
Ah sorry, auto save tricked me